tasks/pre_commit.py

```diff
@@ -0,0 +1,20 @@
+from invoke import task
+from invoke.context import Context
+
+from tasks.libs.common.color import color_message
+
+
+@task()
+def shell_check_no_set_x(ctx: Context):
+    """
+    Check that shell scripts do not use 'set -x'"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**  Could you please add the xtrace one as well ?